calls.go

```diff
@@ -82,14 +106,42 @@ func (call *callsCmd) list(ctx *cli.Context) error {
 		App:     app,
 		Context: context.Background(),
 	}
-	if ctx.Args().Get(1) != "" {
-		route := ctx.Args().Get(1)
-		params.Route = &route
+	if ctx.String("cursor") != "" {
+		cursor := ctx.String("cursor")
+		params.Cursor = &cursor
 	}
+	if ctx.String("path") != "" {
+		route := ctx.String("path")
+		params.Path = &route
+	}
+	if ctx.String("from-time") != "" {
+		fromTime := ctx.String("from-time")
+		fromTime_int64, err := time.Parse(time.RFC3339, fromTime)
```

Comment:
`date --rfc-3339=date` will generate these, but I believe we need the nanosecond version and users may trip up on this. taking `unix` time here (as the api does) is just an int and may be less error prone.  thoughts ?

Reply:
each, `Unix` would be fine, I thought that nanoseconds would work, but they don't, we expect timestamps to be an int64 representation of timestamp in seconds.

anyways, PR updated.